saleor/static/dashboard-next/orders/components/OrderAddressEditDialog/OrderAddressEditDialog.tsx

```diff
@@ -28,53 +35,56 @@ interface OrderAddressEditDialogProps {
   onConfirm(data: AddressTypeInput);
 }
 
-const OrderAddressEditDialog: React.StatelessComponent<
-  OrderAddressEditDialogProps
-> = ({
-  address,
-  confirmButtonState,
-  open,
-  errors,
-  variant,
-  countries,
-  onClose,
-  onConfirm
-}) => (
-  <Dialog open={open}>
-    <Form initial={address} errors={errors} onSubmit={onConfirm}>
-      {({ change, data, errors, submit }) => (
-        <>
-          <DialogTitle>
-            {variant === "billing"
-              ? i18n.t("Edit billing address", { context: "title" })
-              : i18n.t("Edit shipping address", { context: "title" })}
-          </DialogTitle>
-          <DialogContent>
-            <AddressEdit
-              countries={countries}
-              data={data}
-              errors={errors}
-              onChange={change}
-            />
-          </DialogContent>
-          <DialogActions>
-            <Button onClick={onClose}>
-              {i18n.t("Cancel", { context: "button" })}
-            </Button>
-            <ConfirmButton
-              transitionState={confirmButtonState}
-              color="primary"
-              variant="contained"
-              onClick={submit}
-              type="submit"
-            >
-              {i18n.t("Confirm", { context: "button" })}
-            </ConfirmButton>
-          </DialogActions>
-        </>
-      )}
-    </Form>
-  </Dialog>
+const OrderAddressEditDialog = withStyles(styles, {
+  name: "OrderAddressEditDialog"
+})(
+  ({
+    address,
+    classes,
+    confirmButtonState,
+    open,
+    errors,
+    variant,
+    countries,
+    onClose,
+    onConfirm
+  }: OrderAddressEditDialogProps) => (
+    <Dialog open={open} classes={{ paper: classes.overflow }}>
+      <Form initial={address} errors={errors} onSubmit={onConfirm}>
+        {({ change, data, errors, submit }) => (
+          <>
+            <DialogTitle>
+              {variant === "billing"
+                ? i18n.t("Edit billing address", { context: "title" })
+                : i18n.t("Edit shipping address", { context: "title" })}
```

Comment:
You can simply toggle `billing/shipping` text

Reply:
@piotrgrundas Not really, those are translated and you need to translate entire sentences.